src/components/Regions.js

```diff
@@ -0,0 +1,52 @@
+import { faArrowAltCircleRight } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router-dom';
+import numberFormater from './FormatNumber';
+
+const Regions = () => {
+  const data = useSelector((state) => state.countries.data);
```

Comment:
- [ ] <b>REQUIRED:</b> You can also use `shallowEqual` here as I suggested in the previous comment.